templates/scenarios/js/non-sso-tab/.webappignore

```diff
@@ -0,0 +1,26 @@
+.webappignore
+.fx
+.deployment
+.localSettings
+.vscode
+*.js.map
+*.ts.map
+*.ts
+.git*
+.tsbuildinfo
+CHANGELOG.md
+readme.md
+local.settings.json
```

Comment:
I guess local.settings.json is for Az Func, can you help check if it is still valid for this tab scenario?

Reply:
I copy this file from JavaScript default bot template. I think we don't need to maintain many versions of .webappignore file. So let's keep the content.